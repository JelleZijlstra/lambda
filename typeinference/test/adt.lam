type foo = Bar int | Baz bool in
Bar 3
