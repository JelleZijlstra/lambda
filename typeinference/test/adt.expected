%%stdout%%
Result: module
	let __result__ = (Bar 3)
end
%%stderr%%
